src/client/tweetAnalysis.js

```diff
@@ -38,18 +65,26 @@ function wordCountToSortedList(wordCounts) {
     }
   }
 
-  result.sort((a, b) => { return a.count < b.count; });
+  result.sort((wordCount1, wordCount2) => { return wordCount1.count < wordCount2.count; });
   return result;
 };
 
-module.exports = {
-  /**
-   * Given a list of Tweet objects, return a sorted list of the most frequent words in them
-   * @param tweets
-   * @returns {*}
-   */
-  mostFrequentWords: function (tweets) {
-    const wordCounts = countWords(tweets);
-    return wordCountToSortedList(wordCounts);
-  },
-};
+/**
+ * Group a bunch of tweets by their Tweeter.
+ * @param tweets
+ * @returns {{}}

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['tweets' is defined but never used.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2158438031/issues/source?bid=3148782&fileBranchId=3159641#l77)